nbhttp/websocket/dialer.go

```diff
@@ -155,6 +155,9 @@ func (d *Dialer) DialContext(ctx context.Context, urlStr string, requestHeader h
 		}
 
 		remoteCompressionEnabled := false
+		if resp == nil {
+			panic("response should not be nil")
+		}
```

Comment:
We should not panic except we got an error during the server init period.

Reply:
right, it should not get here. I just put it there for debugging. err == nil and resp == nil should not happen